src/main/java/com/k3ntako/HTTPServer/RouteRegistrar.java

```diff
@@ -5,18 +5,20 @@
 public class RouteRegistrar {
   private RouteRegistry routeRegistry;
   private FileIOInterface fileIO;
+  private UUIDInterface uuid;
 
-  public RouteRegistrar(RouteRegistry routeRegistry, FileIOInterface fileIO) {
+  public RouteRegistrar(RouteRegistry routeRegistry, FileIOInterface fileIO, UUIDInterface uuid) {
     this.routeRegistry = routeRegistry;
     this.fileIO = fileIO;
+    this.uuid = uuid;
   }
 
   public RouteRegistry registerRoutes() throws Exception {
     routeRegistry.registerRoute("GET","/simple_get", new SimpleGet());
     routeRegistry.registerRoute("GET","/simple_get_with_body", new SimpleGetWithBody());
     routeRegistry.registerRoute("GET","/admin", new Admin());
     routeRegistry.registerRoute("GET","/account", new Account());
-    routeRegistry.registerRoute("POST","/simple_post", new SimplePost(fileIO));
+    routeRegistry.registerRoute("POST","/simple_post", new SimplePost(fileIO, uuid));
```

Comment:
Here we want to associate `/simple_post` with a specific fucntion on this class(I made another comment that we should rename this `Reminders` since that is the business logic associate with it). If we follow CRUD conventions that function would be called Create.